doc/book/cookbook/using-routed-middleware-class-as-controller.md

```diff
@@ -201,10 +201,10 @@ class AlbumPage extends AbstractPage
 >         $action = $request->getAttribute('action', 'index') . 'Action';
 > 
 >         if (! method_exists($this, $action)) {
->             return $response->withStatus(404);
+>             return new EmptyResponse(404);
```

Comment:
Want to go ahead and use 
`use Fig\Http\Message\StatusCodeInterface as StatusCode;`
and 
`StatusCode::STATUS_NOT_FOUND` for the 404? Yes it's just docs, but since the rest of the code uses it...